blockchain/blockdao/blob_store.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2024 IoTeX Foundation
+// This source code is provided 'as is' and no warranties are given as to title or non-infringement, merchantability
+// or fitness for purpose and, to the extent permitted by law, all liability for your use of the code is disclaimed.
+// This source code is governed by Apache License 2.0 that can be found in the LICENSE file.
+
+package blockdao
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/iotexproject/go-pkgs/hash"
+
+	"github.com/iotexproject/iotex-core/blockchain/block"
+)
+
+type (
+	BlobStore interface {
+		Start(context.Context) error
+		Stop(context.Context) error
+		BlobExpired(uint64, uint64) bool
```

Comment:
what does the two params means?

Reply:
means the request height and current tip height, if tip - requested > 18 days, it is expired